DQM/TrackingMonitorSource/python/TrackingSourceConfig_Tier0_cff.py

```diff
@@ -29,9 +29,9 @@
 #TrackEffMon_bhmuon.AlgoName                        = 'BHMuonTk'
 #TrackEffMon_bhmuon.FolderName                      = 'Tracking/TrackParameters/TrackEfficiency'
 
-# Split Tracking
-from  DQM.TrackingMonitor.TrackSplittingMonitor_cfi import *
-TrackSplitMonitor.FolderName = 'Tracking/TrackParameters/SplitTracks'
+# # Split Tracking
```

Comment:
```suggestion
# Split Tracking
```
(not important, but if you have to modify anyhow...)

Reply:
not sure to see why that would be needed.